Nos/Views/NoteButton.swift

```diff
@@ -13,8 +13,8 @@ struct NoteButton: View {
     var style = CardStyle.compact
     var shouldTruncate: Bool
     var hideOutOfNetwork: Bool
-    var showReplyCount: Bool
-    var displayRootMessage: Bool 
+    var replyCount: ReplyCount?
```

Comment:
```suggestion
    var repliesDisplayType: RepliesDisplayType?
```

Obviously I'd also update the `init` parameter to match.